### ISSUE TITLE
FSR 2.2

### ISSUE THREAD

**Issue by MOVZX** [GPUOpen-Effects/FidelityFX-FSR2#75]
FSR 2.2 has just been announced, where is the source?

**Comment by MOVZX** [comment]
We're very excited to the latest AMD technology.

**Comment by 0x-FADED** [comment]
@rys my man release FSR2.2 source pl0x 🙏🙏🙏🙏

**Comment by imniko** [comment]
WhEn FsR 2point2?

**Comment by 0x-FADED** [comment]
> We're very excited to the latest AMD technology.

Yeah AMD is the best for making their tech open source for everyone to use

**Comment by MOVZX** [comment]
> > We're very excited to the latest AMD technology.
> 
> Yeah AMD is the best for making their tech open source for everyone to use

FSR is the real saviour for PC gaming industry.

**Comment by 0x-FADED** [comment]
@rys and the AMD team thank you ♥️

**Comment by rys** [comment]
We're working on finishing off the public release and it'll be out ASAP. Keep an eye on GPUOpen for the news as soon as we're done 🤓

**Comment by kassindornelles** [comment]
> FSR 2.2 has just been announced, where is the source?

Your profile picture gives me PTSD please change it.

Btw
Also, really excited to see what changes in 2.2
2.1 was already massive, can't wait to see the limits of fsr 2 without having to make it incompatible with older GPU's because of a possible AI hw accelerator requirement

C'mon rys, we are freaking out gib fsr 2.2 sauce code UwU

**Comment by stefan998** [comment]
@rys Is it possible for you to implement Native mode to FSR 2.2 (rendering with 100% resolution scale and only applying FSR anti-aliasing effect), so we can use it in Unreal Engine via console command?

**Comment by MOVZX** [comment]
_"My boah..."_
_"FSR 2.2 is nowhere to be found."_

**Arthur Morgan**

**Comment by imniko** [comment]
FSR 2.2 needs a respin to fix the issue preventing to reach the desired image quality target at reasonable performance. Software team failed this patch.

**Comment by imniko** [comment]
At this rate we will get FSR 2.3 before FSR 2.2

**Comment by MOVZX** [comment]
Mama Su said in presentation video, it'll be FSR 3, and not 2.3.

**Comment by MOVZX** [comment]
@rys Are you still working with AMD man? We miss you...

**Comment by rys** [comment]
> @rys Are you still working with AMD man? We miss you...

We're still working on it, no new comment on when though other than it's still happening as soon as possible. Thank you for hanging in there 👍

**Comment by imniko** [comment]
Please take all the time you need, but do add a better debug options like DLSS or XESS.

**Comment by iNiKKo** [comment]
FSR 2 for UE 5.1 ?

**Comment by atomizasser** [comment]
When is expected release update FSR 2.2 source?

**Comment by imniko** [comment]
2.2 is so last year, they will prepare 2.3 for 2023 release.

**Comment by BenceCsizmadia** [comment]
Any ETA for FSR 2.2?

**Comment by BenceCsizmadia** [comment]
@rys 
Any chance just for an ETA or any update about the plugin?

**Comment by imniko** [comment]
Its been 4 months since no new FSR2 version out yet, its crazy to think that FSR2 would die just like that.

**Comment by MOVZX** [comment]
Guys, I'm pretty sure FSR 2.2 will be released after 2.3 is out.

**Comment by TheRazerMD** [comment]
@MOVZX Soon™

**Comment by imniko** [comment]
Finally this can be closed.
